### PR TITLE
Fix links that are broken

### PR DESCRIPTION
**Pull request by harikt**
I have currently pointed to latest. May be I should specifically point to the version ?

Thank you.

### CODE REVIEW

File: docs/en/tutorials/getting-started.rst [hunk 1]
```diff
@@ -157,12 +157,12 @@ The second block consists of the instantiation of the ORM
 ``Configuration`` object using the Setup helper. It assumes a bunch
 of defaults that you don't have to bother about for now. You can
 read up on the configuration details in the
-`reference chapter on configuration <../reference/configuration.rst>`_.
+`reference chapter on configuration <http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/reference/configuration.html>`_.
```
Comment (Ocramius) [inline docs/en/tutorials/getting-started.rst]: This one should probably be an relative RST internal link target: http://docutils.sourceforge.net/docs/user/rst/quickref.html#internal-hyperlink-targets

Comment (harikt) [inline docs/en/tutorials/getting-started.rst]: What is the point of internal link in a website ?

Comment (Ocramius) [inline docs/en/tutorials/getting-started.rst]: RST doesn't just compile to HTML ;-)

Comment (harikt) [inline docs/en/tutorials/getting-started.rst]: So I am confused or don't know what to do. Feel free to make the changes or I can remove the change made.

**Comment by harikt** [comment]
@Ocramius what should be done in this PR to get this merged ?

**Comment by Ocramius** [comment]
The link should become an internal RST reference. I linked the relevant docs, but you basically need to compile on your side (I suggest compiling the PDF version) to see if it is working

**Comment by harikt** [comment]
That actually brings 2 questions

1.  The links in the website will not work.
2. I am not aware of building pdf from sphinx. If there is some quick stuffs that does it will be helpful.

**Comment by Ocramius** [comment]
@mikeSimonson maybe you can help further here?

**Comment by mikeSimonson** [comment]
@Ocramius I first have to finish the fix for the api links and then I will look at this one.

**Pull request closed without merging**